### ISSUE TITLE
dismissKeyboard() method?

### ISSUE THREAD

**Issue by gitasong** [reactnativecn/react-native-custom-keyboard#16]
What's the possibility of having a `dismissKeyboard()` method written for this module? I know that the original author is no longer actively developing this, but that others are. 

I'd really like to use this module in a project—it's simple and elegant—but I don't write any Java myself. Is there anyone who'd be willing to add this method?

**Comment by sidalihallak** [comment]
import { Keyboard } from 'react-native';

Keyboard.dismiss()

**Comment by gitasong** [comment]
> import { Keyboard } from 'react-native';
>Keyboard.dismiss()

Yeah—I finally figured that out. It wasn't obvious from the code that the custom keyboard was being registered as an actual Android keyboard, so I didn't think I could actually use this. Thanks for the confirmation! It would be nice if this could be included in the README so that junior folks like me wouldn't have to hunt for it. :)
